src/Microsoft.TestPlatform.Common/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -250,7 +250,7 @@ static Microsoft.VisualStudio.TestPlatform.Common.SettingsProvider.SettingsProvi
 static Microsoft.VisualStudio.TestPlatform.Common.SettingsProvider.SettingsProviderExtensionManager.LoadAndInitializeAllExtensions(bool shouldThrowOnError) -> void
 static Microsoft.VisualStudio.TestPlatform.Common.Utilities.CancellationTokenExtensions.ThrowTestPlatformExceptionIfCancellationRequested(this System.Threading.CancellationToken token) -> void
 static Microsoft.VisualStudio.TestPlatform.Common.Utilities.ExceptionUtilities.GetExceptionMessage(System.Exception? exception) -> string!
-static Microsoft.VisualStudio.TestPlatform.Common.Utilities.FakesUtilities.GenerateFakesSettingsForRunConfiguration(string![]! sources, string! runSettingsXml) -> string!
+static Microsoft.VisualStudio.TestPlatform.Common.Utilities.FakesUtilities.GenerateFakesSettingsForRunConfiguration(System.Collections.Generic.IDictionary<string!, Microsoft.VisualStudio.TestPlatform.ObjectModel.Framework!>! sourceToFrameworkMap, string! runSettingsXml) -> string!
```

Comment:
💭 Just some open question: Could we make some internals visible to for Test Window if we except them to be the only consumers rather than having something public? It's always weird to me to see us doing "breaking change" in the middle of a version.